backend/src/main/java/corea/feedback/service/RevieweeToReviewerFeedbackService.java

```diff
@@ -0,0 +1,68 @@
+package corea.feedback.service;
+
+import corea.exception.CoreaException;
+import corea.exception.ExceptionType;
+import corea.feedback.domain.RevieweeToReviewerFeedback;
+import corea.feedback.dto.RevieweeToReviewerFeedbackRequest;
+import corea.feedback.dto.RevieweeToReviewerResponse;
+import corea.feedback.repository.RevieweeToReviewerFeedbackRepository;
+import corea.matching.domain.MatchResult;
+import corea.matching.repository.MatchResultRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class RevieweeToReviewerFeedbackService {
+
+    private final RevieweeToReviewerFeedbackRepository revieweeToReviewerFeedbackRepository;
+    private final MatchResultRepository matchResultRepository;
+
+    @Transactional
+    public RevieweeToReviewerResponse create(long roomId, long revieweeId, RevieweeToReviewerFeedbackRequest request) {
+        validateAlreadyExist(roomId, request.reviewerId(), revieweeId);
+        RevieweeToReviewerFeedback revieweeToReviewerFeedback = revieweeToReviewerFeedbackRepository.save(createEntity(roomId, revieweeId, request));
+        return RevieweeToReviewerResponse.of(revieweeToReviewerFeedback);
+    }
+
+    @Transactional
+    public RevieweeToReviewerResponse update(long feedbackId, long roomId, long revieweeId, RevieweeToReviewerFeedbackRequest request) {
+        validateNotExist(feedbackId);
+        RevieweeToReviewerFeedback revieweeToReviewerFeedback = revieweeToReviewerFeedbackRepository.save(createEntity(feedbackId, roomId, revieweeId, request));
+        return RevieweeToReviewerResponse.of(revieweeToReviewerFeedback);
+    }
+
+    public RevieweeToReviewerResponse findRevieweeToReviewer(long roomId, long revieweeId, String username) {
```

Comment:
[제안] 이 메소드 뒤에 feedback이라는 단어 추가하시는 건 어떤가요? 

```suggestion
    public RevieweeToReviewerResponse findRevieweeToReviewerFeedback(long roomId, long revieweeId, String username) {
```

Reviewer->Reviewee Service도 동일합니다.

Reply:
반영했습니다~